README.md

```diff
@@ -1,13 +1,11 @@
 # Project Name
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+For this week’s project we were to create our own digital version of the classic board game Guess who, practising objects and arrays and connecting them to the DOM. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I watched provided videos, did Sandbox practices, looked at provided code and compared to example code. I had a hard time this week finding enough time to practice and difficulties getting started on the project. My plan was to keep to practising the first two days of the week and start building the project after that. The plan was impossible to keep to since it all became a bit overwhelming, leaving me at a loss and a bit incapacitated as to where to begin, how to approach the task. I had to decide on copying and tweaking someone else's code as a way of learning, as I've found quite helpful before, or building my own code as far as I've been able to come this week. I chose the latter, which unfortunately leaves me with an unfinished project, but at least I understand each step so far. If I had more time I would have firstly gotten everything to work and then created my own version of the project, which would have added some joy to learning all of this. Again, this has been a disheartening process in regards to how much is asked of us and how little I've been able to accomplish – but I still look forward to learning all of this and that leaves me in good spirits.
```

Comment:
You're not alone! Remember to reach out to your team for support and I'm glad to hear you're still in good spirits to keep learning ❤️ #trustTheProcess